### PR TITLE
add code

### PR DESCRIPTION
**Pull request by stevelee2497**

### CODE REVIEW

File: Exercise7/Controllers/Adapters/CountryAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        private const int TypeNormal = 1, TypeHead = 2;
+
+        private List<Country> countries;
+
+        public CountryAdapter(List<Country> countries)
+        {
+            this.countries = countries;
+        }
+
+        public override int ItemCount => countries.Count;
```
Comment (voquanghoa) [inline Exercise7/Controllers/Adapters/CountryAdapter.cs:19]: The number of view items should be 
```C#
countries.Count + countries.Select(x=>x.Name.First()).Distinct().Count;
```

File: Exercise2/Models/Adapters/ProductAdapter/ProductAdapter.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ namespace Exercise2.Models.Adapters.ProductAdapter
 {
     class ProductAdapter :RecyclerView.Adapter
     {
-        private List<Product> products;
+        private readonly List<Product> products;
```
Comment (voquanghoa) [inline Exercise2/Models/Adapters/ProductAdapter/ProductAdapter.cs:9]: Unused ??

File: Exercise7/Controllers/Adapters/CountryAdapter.cs (hunk 1 above)
Comment (stevelee2497) [reply Exercise7/Controllers/Adapters/CountryAdapter.cs:19]: I don't think so. It's just 2 type of view holder : 1st type is the country name and 2nd type contain both country name and country letter so it's just countries.Count.

File: Exercise2/Models/Adapters/ProductAdapter/ProductAdapter.cs (hunk 2 above)
Comment (stevelee2497) [reply Exercise2/Models/Adapters/ProductAdapter/ProductAdapter.cs:9]: yah, i'll delete it

File: Exercise7/Controllers/Adapters/CountryAdapter.cs (hunk 1 above)
Comment (voquanghoa) [reply Exercise7/Controllers/Adapters/CountryAdapter.cs:19]: Ok fine.

**Pull request merged**